Library/Homebrew/hardware.rb

```diff
@@ -55,6 +55,8 @@ def self.cores_as_words
     when 1 then "single"
     when 2 then "dual"
     when 4 then "quad"
+    when 6 then "hexa"
```

Comment:
Maybe just `six` and `eight` for these; I don't tend to hear `octacore` used and it's probably less obvious to non-europeans?